### ISSUE TITLE
Unusual accessibility requirements mappings

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wcag-act#478]
The [text has minimal contrast](https://act-rules.github.io/rules/afw4f7) rule maps both to SC 1.4.3 contrast minimum, and to SC 1.4.6 contrast enhanced. This seems unintuitive, because the text-contrast rule checks for a contrast ratio of 4.5:1, not 7:1 like 1.4.6 requires. The reason this mapping was added is because failing to meet a contrast ratio of 4.5:1 also results in a failure of 7:1. This was done because the ACT Rules Format has the following requirement:

> [4.4. Accessibility Requirements Mapping
](https://www.w3.org/TR/act-rules-format/#accessibility-requirements-mapping)
> 
> When an ACT Rule is designed to test conformance to one or more Accessibility requirements documents, the rule must list all accessibility requirements from those documents that are not satisfied when one or more of the outcomes of the rule is failed

In other words, because a text that fails a 4.5:1 contrast ratio also fails a 7:1 contrast ratio, the mapping **must** include both SCs. This makes sense logically, but could be a little confusing. An important thing to note about that is that some of the "passed" test cases in the text-contrast rule do not meet that 7:1 contrast ratio. This is unusual for rules, where generally passed and inapplicable test cases are written in a way that satisfies the SC.

There are other SCs too, where we have rules that have this unusual mapping in them:
-  1.2.3 Audio Description or Media Alternative (Prerecorded)
    - 1.2.5 Audio Description (Prerecorded)
    - 1.2.8 Media Alternative (Prerecorded)
- 1.4.4 Resize text
    - 1.4.10 Reflow
- 2.1.1 Keyboard
    - 2.1.3 Keyboard (No Exception)
- 2.2.1 Timing Adjustable
    - 2.2.3 No Timing
    - 2.2.6 Timeouts
- 2.3.1 Three Flashes or Below Threshold
    - 2.3.2 Three Flashes
- 2.4.4 Link Purpose (In Context)
    - 2.4.9 Link Purpose (Link Only)
- 3.2.1 On Focus
    - 3.2.5 Change on Request
- 3.2.2 On Input
    - 3.2.5 Change on Request

Thinking about this a little more closely, it seems to me that having this mapping is useful. I don't think we should take them out, but I do think there's something we can do to improve the presentation to make it clearer that there is something different about that particular requirement.

**Comment by WilcoFiers** [comment]
Since this is blocking one of our rules, I'm going to try to use the background section to address this. We can work on a solution that uses the requirements mapping later.

**Comment by nitedog** [comment]
I think we should have a label for "future spec improvements" or such, to consider in a future update of the spec.

**Comment by WilcoFiers** [comment]
Just so there's a record of this. The community group is currently putting information about these "incidentally supported" requirements in the background section. This seems to work quite well. I'm going to leave the ticket open, as this is something we probably want to revisit in an ACT Rules Format update.

**Comment by kengdoj** [comment]
I think Secondary Requirements addresses this.
